src/components/trello/style.scss

```
@@ -0,0 +1,106 @@
+.container {
+  max-width: 1400px;
+  margin: 0 auto;
+  padding: 0 5px;
+}
+.main {
+  background-color: #2c3e50;
+  width: 100%;
+  height: 100vh;
+  padding-top: 10px;
+
+  &__grid {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+  }
+
+  &__card {
+    background-color: #a7bfe8;
+    max-width: 330px;
+    max-height: 800px;
+    overflow-y: auto;
+  }
+
+  &__textarea {
+    resize: none;
+    max-width: 280px;
+    height: 70px;
+    font-size: 20px;
+  }
+
+  &__click_add {
+    margin-top: 20px;
+    padding-left: 10px;
+    width: 200px;
+    height: 30px;
+    display: flex;
+    column-gap: 6px;
+    &:hover {
+      width: 200px;
+      height: 30px;
+      border-radius: 5px;
+      background-color: white;
+    }
+  }
+
+  &__save {
+    display: flex;
+    row-gap: 10px;
+    flex-direction: column;
+  }
+
+  &__save_gap {
+    display: flex;
+    column-gap: 10px;
+  }
+
+  &__text {
+    position: fixed;
+    margin-bottom: 10px;
+    width: 100%;
+    height: 40px;
+  }
+
+  &__click_button {
+    cursor: pointer;
+  }
+
+  &__button {
+    cursor: pointer;
+  }
+
+  &__card_2 {
```

При названии классов не стоит использовать цифры, это не очень хорошо 
```suggestion
  &__first-card {
```